### ISSUE TITLE
Specify order of options when there are global and command options

### ISSUE THREAD

**Issue by dlong500** [yargs/yargs#1181]
Right now when showing help it appears that any global options appear first followed by options defined within a command.

Is there a way to make it so that the command options appear first followed by the global options?  It could even be useful to allow some global options to appear before the command options and have the rest appear after, but at least for now I'd settle on have all global options follow the command options.

**Comment by teppeis** [comment]
Workaround: `.alias('v', 'version')` (not `.alias('version', 'v')`) shows `--version` option at the bottom.

- [Example code](https://github.com/teppeis/duck/blob/4fec51029e6fa475bb550a00789a2b2c86cb1fba/src/cli.ts#L305-L306)
- [The output](https://github.com/teppeis/duck/blob/4fec51029e6fa475bb550a00789a2b2c86cb1fba/README.md#duck-serve)

**Comment by bcoe** [comment]
@dlong500 @teppeis seems reasonable to me that we'd make the command specific options show up before the global options; perhaps keep this open as a feature request?

**Comment by bcoe** [comment]
I don't think we should go as far as allowing this to be configurable, it gets in the territory of adding to many configuration options for my liking.
